### PR TITLE
stripe-cli 1.19.5 (new formula)

### PR DESCRIPTION
**Pull request by carlocab**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

File: Formula/s/stripe-cli.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    bash_completion.install "stripe-completion.bash"
+    zsh_completion.install "stripe-completion.zsh"
+    (zsh_completion/"_stripe").write <<~ZSH
+      #compdef stripe
+      _stripe () {
+        local e
+        e=$(dirname ${funcsourcetrace[1]%:*})/stripe-completion.zsh
+        if [[ -f $e ]]; then source $e; fi
+      }
+    ZSH
```
Comment (SMillerDev) [inline Formula/s/stripe-cli.rb:29]: Wouldn't we be fine if we rename the `stripe-completion.zsh` to `_stripe`?

Comment (carlocab) [reply Formula/s/stripe-cli.rb]: Didn't work for some reason when I first tried it, but it seems to be working now 🤷

File: Formula/s/stripe-cli.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    ENV["CGO_ENABLED"] = OS.mac? ? "1" : "0"
+    ldflags = %W[-s -w -X github.com/stripe/stripe-cli/pkg/version.Version=#{version}]
+    system "go", "build", *std_go_args(ldflags:, output: bin/"stripe"), "cmd/stripe/main.go"
+
+    # Doesn't work with `generate_completions_from_executable`
+    # Taken from `.goreleaser` directory
+    system bin/"stripe", "completion", "--shell", "bash"
+    system bin/"stripe", "completion", "--shell", "zsh"
+    bash_completion.install "stripe-completion.bash"
+    zsh_completion.install "stripe-completion.zsh" => "_stripe"
```
Comment (chenrui333) [inline Formula/s/stripe-cli.rb:21]: this should work
```suggestion
    generate_completions_from_executable(bin/"stripe", "completion", 
                                            shells: [:bash, :zsh], 
                                            shell_parameter_format: :none, 
                                            base_name: "stripe")
```

Comment (chenrui333) [reply Formula/s/stripe-cli.rb:21]: indeed 😅

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**